lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/EvaluatorRequestorImpl.java

```diff
@@ -63,6 +63,10 @@ public synchronized void submit(final EvaluatorRequest req) {
     LOG.log(Level.FINEST, "Got an EvaluatorRequest: number: {0}, memory = {1}, cores = {2}.",
         new Object[] {req.getNumber(), req.getMegaBytes(), req.getNumberOfCores()});
 
+    for (String nodeName : req.getNodeNames()) {
```

Comment:
Apache Commons has code for this which you can use to format this into a single log line.